selfdrive/modeld/models/commonmodel.c

```diff
@@ -66,3 +66,11 @@ float sigmoid(float input) {
 float softplus(float input) {
   return log1p(expf(input));
 }
+
+float input_lambda(float input) {
```

Comment:
Is this function being inlined? I'd use a macro here.

Reply:
macro it is